Marlin/SanityCheck.h

```diff
@@ -292,7 +292,15 @@
   #endif
 
   /**
+   * Check if ENDSTOPS_ONLY_FOR_HOMING is disabled when M211 command is enabled
+   */
+  #if (defined(ENABLE_M211)) && (!defined(ENDSTOPS_ONLY_FOR_HOMING))
+    #error ENDSTOPS_ONLY_FOR_HOMING must be enabled when M211 command is active.
+  #endif
+
+  /**
    * Warnings for old configurations
+   * Check if ENDSTOPS_ONLY_FOR_HOMING is disabled when M211 command is enabled
```

Comment:
> In Marlin/SanityCheck.h
> <https://github.com/MarlinFirmware/Marlin/pull/1814#discussion_r28127276>:
>
> >     * Warnings for old configurations
> > +   * Check if ENDSTOPS_ONLY_FOR_HOMING is disabled when M211 command is enabled
>
> Or you can just set Z to 10 by G92 Z10. If it's at zero you can now go
> down by 10mm.
>
That's an option but needs g-code dealing between 0mm and 10mm in
difference to g-code working between -10mm and 0mm.

This command fulfil my requirements. If there is a wish to integrate
this command in marlin i clean up the code. Other way i let this code in
my fork of marlin.